lib/client.js

```diff
@@ -1,11 +1,38 @@
 var PROMISE_TIMEOUT = 5000, // 5 seconds
     ZAF_EVENT       = /^zaf\./,
     version         = require('version'),
+    Utils           = require('utils'),
     Promise         = window.Promise || require('../vendor/native-promise-only'),
     when            = require('utils').when,
     pendingPromises = {},
     ids             = {};
 
+// @internal
+// #timeoutReject(name, params)
+//
+// Reject a request if required, given the rejection function, name (get, set or invoke)
+// and arguments to that request. Falls back to 5000 second timeout with few exceptions.
+//
+function timeoutReject(reject, name, paramsArray) {
+  switch(name) {
+  case 'invoke': {
```

Comment:
Can we change the config? I find it pretty weird to have the two closing brackets on the same indentation level below. 😛